### PR TITLE
All namespaces and headers that use transparent_auth to authservice

### LINKED ISSUE #45: Rename internal top level namespace from transparent_auth to authservice
Rename internal top level namespace from transparent_auth to authservice to match repository name.

### PR DESCRIPTION
**Pull request by neilong31**
Fixes #45 
Any time I saw transparent_auth as a header, namespace, or in the comments I changed it to authservice in both the src and test folders. I also followed the same coding conventions.

### CODE REVIEW

**Comment by neilong31** [comment]
Thanks! I have signed-off on my commit and filled out my google CLA.

File: src/config/getconfig.cc [hunk 1]
```diff
@@ -11,8 +11,8 @@ using namespace std;
 using namespace google::protobuf::util;
 using namespace authservice::config;
 
-// TODO: s/namespace transparent_auth/namespace authservice/g
-namespace transparent_auth {
+// TODO: s/namespace authservice/namespace authservice/g
```
Comment (nickrmc83) [inline src/config/getconfig.cc]: Can you remove this TODO. You've just made the changes it describes are required.

Comment (neilong31) [reply src/config/getconfig.cc]: No problem! Should have just deleted it as I was going through the code.

**Pull request merged**